src/components/progressCircular/js/progressCircularDirective.js

```diff
@@ -200,7 +197,9 @@ function MdProgressCircularDirective($$rAF, $window, $mdProgressCircular, $mdThe
       if (animateTo === animateFrom) {
         path.attr('d', getSvgArc(animateTo, diameter, pathDiameter, rotation));
       } else {
-        lastDrawFrame = $$rAF(function animation(now) {
+        // Note that this shouldn't use use $$rAF, because it can cause an infinite loop
+        // in any tests that call $animate.flush.
+        $window.requestAnimationFrame(function animation(now) {
```

Comment:
Note that the ng1 `RAFProvider` does this:

```js
function $$RAFProvider() { //rAF
  this.$get = ['$window', '$timeout', function($window, $timeout) {
    var requestAnimationFrame = $window.requestAnimationFrame ||
                                $window.webkitRequestAnimationFrame;

    var cancelAnimationFrame = $window.cancelAnimationFrame ||
                               $window.webkitCancelAnimationFrame ||
                               $window.webkitCancelRequestAnimationFrame;

    var rafSupported = !!requestAnimationFrame;
```

With the can on L202, we lose all of that! My first thought is that this is the wrong solution.